codespell_lib/data/dictionary.txt

```diff
@@ -7830,12 +7841,15 @@ intiailizing->initializing
 intial->initial
 intialisation->initialisation
 intialise->initialise
-intialised->initialized
-intialises->initializes
+intialised->initialised
+intialiser->initialiser
+intialises->initialises
+intialises->initialises
```

Comment:
From Travis

u'intialises' already exists

Reply:
That's not sure the problem -- it's complaining that there is now a duplicate entry. It's in there twice if you look at the diff (here).

Reply:
Lines 7846 and 7847 are identical in this case, delete one and you should be good